### PR TITLE
fixes broken logo link in communities.json

### PR DESCRIPTION
**Pull request by notwaving**
Defect #10

### CODE REVIEW

File: data/communities.json [hunk 1]
```diff
@@ -108,7 +108,7 @@
     "name": "Unfold Inclusion in UK XR",
     "description": "Powering inclusion in immersive tech. We're a support network that actively promotes & champions the inclusion of all underrepresented groups in the VR, AR and MR space. Formerly Women in VR UK",
     "url": "https://www.facebook.com/groups/unfolduk/",
-    "logo": "https://scontent-otp1-1.xx.fbcdn.net/v/t1.0-9/18485911_10156375476758532_7563742107552031879_n.jpg?oh=072724ab61f485af2fb601d40a0ae0b6&oe=59B3E6A4",
+    "logo": "https://www.meetup.com/unfolduk/photos/26949852/460377120/",
```
Comment (eddiejaoude) [inline data/communities.json]: That link doesn't have a file on the end? Using that link and selecting the first / default photo, I think it is this...

`https://secure.meetupstatic.com/photos/event/d/f/2/0/highres_460377120.jpeg`

Which is this image...
Unfold

Comment (notwaving) [reply data/communities.json]: My coffee's defective. 

*Internal screaming*

PR should be updated now.

Comment (eddiejaoude) [reply data/communities.json]: Happens to the best of us 😄 . Thanks 👍

**Review by eddiejaoude — APPROVED** [review]

**Comment by eddiejaoude** [comment]
Thank you 👍

**Pull request merged**